ext/standard/tests/general_functions/print_r.phpt

```diff
@@ -589,13 +589,13 @@ PHP
 PHP
 PHP
 -- Iteration 12 --
-abcd n1234 05678 00efgh\xijkl
-abcd n1234 05678 00efgh\xijkl
-abcd n1234 05678 00efgh\xijkl
```

Comment:
If you're wondering what's going on here, it appears to be a bug in GitHub's diff viewer. The `00efgh\xijkl` suffix on these lines if present in the old version as well, but GitHub doesn't display it.